packages/form-core/src/FieldApi.ts

```diff
@@ -326,6 +328,13 @@ export class FieldApi<
             ? state.meta.errors
             : []
 
+          const fieldDefaultValue = getBy(
+            this.form.options.defaultValues,
+            this.name,
```

Comment:
Is it OK to assume that passing `this.form.options.defaultValues` as the first argument to `getBy()` will never result in an error? `defaultValues` cannot be `null`,  or any other value that could cause us trouble here, right? 😅